pages/new-project-page.js

```diff
@@ -0,0 +1,5 @@
+import Form from "@/components/Form";
```

Comment:
it is a bit unusual to have a page calles 'new-project-page'. You could rename this file to 'new-project' or 'create-project' which sounds a bit nicer if you see it in the browser address bar.